README.md

```diff
@@ -17,11 +17,13 @@ since 2011, and has grown to encompass tens of thousands of MySQL nodes.
 
 For more about Vitess, please visit [vitess.io](http://vitess.io).
 
+Vitess has a growing community. You can view the list of adopters [here](ADOPTERS.md).
```

Comment:
Fixed. We should work on getting this exposed on vitess. io. But I was thinking the work there may have to be more fancy, with logos, etc.